src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Shapes/Basics_Shapes_Tests.cs

```diff
@@ -0,0 +1,628 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Shapes
+{
+	public class Basics_Shapes_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.iOS)]
+		public void ValidateAllShapesBasicStrechesAlignemntsAndSize()
+		{
+			Run("UITests.Windows_UI_Xaml_Shapes.Basic_Shapes", skipInitialScreenshot: true);
+
+			var expectedDirectory = Path.Combine(TestContext.CurrentContext.TestDirectory, "Windows_UI_Xaml_Shapes/Basics_Shapes_Tests_EpectedResults");
+			var tolerance = new PixelTolerance()
+				.WithColor(96) // We are almost only trying to detect edges
+				.WithOffset(3, 3, LocationToleranceKind.PerPixel)
+				.Discrete(20); // Way toooooooo long otherwise!
+
+			foreach (var test in _tests)
+			{
+				try
+				{
+					var expected = new FileInfo(Path.Combine(expectedDirectory, $"{test}.png"));
+					if (!expected.Exists)
+					{
+						Assert.Fail($"Expected screenshot does not exists ({expected.FullName})");
+					}
+
+					_app.EnterText("_idInput", test);
+					_app.Tap("_renderId");
+
+					var actual = TakeScreenshot(test, ignoreInSnapshotCompare: true);
+					var testZone = _app.WaitForElement("_testZone").Single().Rect;
+					var scale = 2.0;
+
+					ImageAssert.AreAlmostEqual(expected, ImageAssert.FirstQuadrant, actual, testZone.ToRectangle(), scale, tolerance);
+
+					Assert.Fail();
+				} catch(Exception) { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5363624)